moda-contracts/src/Catalog.sol

```diff
@@ -410,7 +411,7 @@ contract Catalog is ICatalog, AccessControlUpgradeable {
     function _requireTrackWritePermissions(address trackOwner, address caller) internal view {
         CatalogStorage storage $ = _getCatalogStorage();
 
-        if (caller != trackOwner && !IModaRegistry($._modaRegistry).isManager(trackOwner, caller)) {
+        if (caller != trackOwner && !IManagement($._management).isManager(trackOwner, caller)) {
```

Comment:
Here we can just call `modaRegistry.getManagement().isManager(trackOwner, caller);`